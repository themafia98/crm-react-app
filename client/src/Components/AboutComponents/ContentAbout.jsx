import React from 'react';
import PropTypes from 'prop-types';
import './content.scss';
const ContentAbout = ({photoUrl, content}) => {

    return (
        <div className = 'About__Content'>
            <div className = 'col-6'>
                <img className = 'myPhoto' 
                    src = {photoUrl}
                    alt = 'myPhoto' 
                />
            </div>
            <div className = 'col-6 About__Content__contentMain'>
                <h3>100 лет опыта в бизнесе и ИТ технологиях</h3>
                <p>{content}</p>
            </div>
        </div>
    )

};

ContentAbout.propsTypes = {
    photoUrl: PropTypes.string.isRequired,
    content: PropTypes.oneOfType([
        PropTypes.string.isRequired,
        PropTypes.array.isRequired
    ])
};

export default ContentAbout;