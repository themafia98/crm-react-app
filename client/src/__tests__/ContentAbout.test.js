import React from 'react';
import renderer from 'react-test-renderer';

import ContentAbout from '../Components/AboutComponents/ContentAbout';

const ContentAboutTest = renderer.create(
    <ContentAbout content = 'Hello test done!' photoUrl = '/img/test.jpg'  />
);

test('ContentAbout test', () => {

    let treeSnapshotContentAbout = ContentAboutTest.toJSON();
    expect(treeSnapshotContentAbout).toMatchSnapshot();
});