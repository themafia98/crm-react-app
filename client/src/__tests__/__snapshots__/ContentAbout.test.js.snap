// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentAbout test 1`] = `
<div
  className="About__Content"
>
  <div
    className="col-6"
  >
    <img
      alt="myPhoto"
      className="myPhoto"
      src="/img/test.jpg"
    />
  </div>
  <div
    className="col-6 About__Content__contentMain"
  >
    <h3>
      100 лет опыта в бизнесе и ИТ технологиях
    </h3>
    <p>
      Hello test done!
    </p>
  </div>
</div>
`;
